lte/gateway/c/session_manager/RedisStoreClient.cpp

```diff
@@ -11,28 +11,28 @@
  * limitations under the License.
  */
 
-#include "RedisStoreClient.h"
-
-#include <cpp_redis/core/client.hpp>  // for client, client::connect_state
-#include <cpp_redis/core/reply.hpp>   // for reply
-#include <cpp_redis/misc/error.hpp>   // for redis_error
-#include <folly/Range.h>              // for operator<<, StringPiece
-#include <folly/dynamic.h>            // for dynamic
-#include <folly/json.h>               // for parseJson, toJson
-#include <glog/logging.h>             // for COMPACT_GOOGLE_LOG_INFO, LogMes...
-#include <stddef.h>                   // for size_t
-#include <stdint.h>                   // for uint32_t
-#include <yaml-cpp/yaml.h>            // IWYU pragma: keep
-#include <future>                     // for future
-#include <ostream>                    // for operator<<, basic_ostream, size_t
-#include <unordered_map>              // for _Node_iterator, unordered_map
-#include <utility>                    // for move, pair
-#include <vector>                     // for vector
-
-#include "SessionState.h"  // for SessionState
-#include "StoredState.h"   // for deserialize_stored_session, ser...
-#include "includes/ServiceConfigLoader.hpp"  // for ServiceConfigLoader
-#include "magma_logging.h"                   // for MERROR, MLOG
+#include "lte/gateway/c/session_manager/RedisStoreClient.h"
+
+#include <cpp_redis/core/client.hpp>
+#include <cpp_redis/core/reply.hpp>
+#include <cpp_redis/misc/error.hpp>
+#include <folly/Range.h>
```

Comment:
The include ordering is currently done by the IWYU script and since there is no automated way of appeasing cpplint right now, I would like to ignore these cpplint warnings for now. The hope is to eventually have an include sorting script that handles this (see https://github.com/magma/magma/issues/12441).

Reply:
Ok Marie! Then its fine.